examples/step-2/doc/intro.dox

```diff
@@ -77,23 +89,54 @@ number of matrix-vector multiplications to come up with the solution of a
 linear system with this matrix, then we would have a solver that can find the
 values of all $N$ unknowns with optimal complexity, i.e., with a total of
 ${\cal O}(N)$ operations. It is clear that this wouldn't be possible if the
-matrix were not sparse, but it also requires very specialized solvers such as
+matrix were not sparse (because then the number of entries in the matrix would
+have to be ${\cal O}(N^s)$ with some $s>1$, and doing a fixed number of
+matrix-vector products would take ${\cal O}(N^s)$ operations), 
+but it also requires very specialized solvers such as
 multigrid methods to satisfy the requirement that the solution requires only a
 fixed number of matrix-vector multiplications. We will frequently look at the
 question of what solver to use in the remaining programs of this tutorial.
 
 The sparsity is generated by the fact that finite element shape
-functions are defined locally on individual cells, rather than
+functions are defined <i>locally</i> on individual cells (or, in some cases such
+as the Discontinuous Galerkin metho, on neighboring cells), rather than
```

Comment:
method

Reply:
I am not sure that I understand this statement about DG methods. Does this state that some functions are defined over multiple cells? That doesn't sound right.

Reply:
@drwells is right that this does not sound fully correct. I would phrase this as "or, in some cases, such as the discontinuous Galerkin methods, also connect to neighboring cells through face integrals"

Reply:
Yes, all good points.